packages/emacs/build.sh

```diff
@@ -122,12 +123,19 @@ termux_step_post_configure() {
 	cp $TERMUX_PKG_HOSTBUILD_DIR/src/bootstrap-emacs $TERMUX_PKG_BUILDDIR/src/bootstrap-emacs
 	cp $TERMUX_PKG_HOSTBUILD_DIR/lib-src/make-docfile $TERMUX_PKG_BUILDDIR/lib-src/make-docfile
 	cp $TERMUX_PKG_HOSTBUILD_DIR/lib-src/make-fingerprint $TERMUX_PKG_BUILDDIR/lib-src/make-fingerprint
+	cp -r $TERMUX_PKG_SRCDIR/lisp/* $TERMUX_PKG_BUILDDIR/lisp
+	cp -r $TERMUX_PKG_SRCDIR/etc $TERMUX_PKG_BUILDDIR
 	# Update timestamps so that the binaries does not get rebuilt:
 	touch -d "next hour" $TERMUX_PKG_BUILDDIR/src/bootstrap-emacs \
 		$TERMUX_PKG_BUILDDIR/lib-src/make-docfile \
 		$TERMUX_PKG_BUILDDIR/lib-src/make-fingerprint
 }
 
+termux_step_make() {
+	make -j $TERMUX_MAKE_PROCESSES
+	mkdir -p $TERMUX_PREFIX/share/emacs/master/lisp/emacs-lisp/
```

Comment:
This looks strange, why master? Tested with master branch of emacs git?

Please also move the mkdir command to `termux_step_post_make_install` to avoid overriding `termux_step_make`